webclient/src/websocket/events/SessionEvents.ts

```diff
@@ -124,7 +124,11 @@ function serverIdentification(info: ServerIdentificationData) {
   switch (webClient.options.reason) {
     case WebSocketConnectReason.LOGIN:
       SessionCommands.updateStatus(StatusEnum.LOGGING_IN, 'Logging In...');
-      SessionCommands.login();
+      if (serverOptions & webClient.protobuf.controller.Event_ServerIdentification.ServerOptions.SupportsPasswordHash) {
```

Comment:
Let's leave a comment here identifying this line as an intentional use of a bitwise operator.